src/fides/api/ctl/migrations/versions/de456534dbda_add_privacyrequest_consent_preferences_.py

```diff
@@ -0,0 +1,43 @@
+"""add privacyrequest consent preferences and ruleuse table
+
+Revision ID: de456534dbda
+Revises: 3caf11127442
+Create Date: 2023-01-03 22:59:45.144538
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "de456534dbda"
+down_revision = "3caf11127442"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "privacyrequest",
+        sa.Column(
+            "executable_consent_preferences",
```

Comment:
This optional list of consent preferences persists to the PrivacyRequest what consent options we're updating, and in particular, just the executable ones.  That way the execution workflow doesn't have to worry about what is and isn't executable, it can use the whole list.